www/src/About.svelte

```diff
@@ -1,11 +1,55 @@
-<h1>About</h1>
+<h1>FAQ</h1>
 
+<h3>Where do I get Görli Ether?</h3>
+<p>
+  Get it from a faucet, or come to our <a
+    href="https://discord.com/invite/GMU7j2kqWQ">discord server</a
+  > and ask for some Göerli Ether.
+</p>
+
+<h3>What are keys and where do I get them?</h3>
+<p>
+  Keys are hidden mnemonic phrases hidden in the game. If you find a key, you
```

Comment:
```suggestion
  Keys are mnemonic phrases hidden in the game. If you find a key, you
```

Reply:
Should we state the length? They are all 24, aren't they?

Reply:
12 and 24